projects/packages/test-environment/src/class-test-environment.php

```diff
@@ -73,7 +75,7 @@ public static function init( $package_slug = null ) {
 				define( 'dbless_UPLOADS', 'uploads-' . $package_slug ); // phpcs:ignore Generic.NamingConventions.UpperCaseConstantName.ConstantNotUpperCase
 			}
 
-			\WorDBless\Load::load();
+			\WorDBless\Load::load( $db_engine, true ); // persist the database across tests so with sqlite tests concurrency is possible without accidental deletions.
```

Comment:
It'll be interesting to see how well that works. It's been several years since I've tried it (and it wasn't in WordPress), but I vaguely remember having locking issues in some cases.

Reply:
Indeed. my ideal would be separate sqlite databases per testing project, but the WP plugin doesn't support custom db file names yet.